x/plans/keeper/keeper.go

```diff
@@ -25,16 +27,18 @@ func NewKeeper(
 	storeKey,
 	memKey sdk.StoreKey,
 	ps paramtypes.Subspace,
+	projectsKeeper types.ProjectsKeeper,
 ) *Keeper {
 	// set KeyTable if it has not already been set
 	if !ps.HasKeyTable() {
 		ps = ps.WithKeyTable(types.ParamKeyTable())
 	}
 
 	return &Keeper{
-		memKey:     memKey,
-		paramstore: ps,
-		plansFs:    *common.NewFixationStore(storeKey, cdc, types.PlanFixationStorePrefix),
+		memKey:         memKey,
```

Comment:
why does plan module know the project module?

Reply:
for the chain policy validation

Reply:
if we remove the spec validation this is not needed

Reply:
done